api/project.go

```diff
@@ -65,47 +65,47 @@ const (
 )
 
 // ProjectRaw is the store model for a project.
+// Fields have exactly the same meanings as Project.
 type ProjectRaw struct {
 	ID int
 
-	// Standard fields
```

Comment:
nit: I prefer to keep Standard fields and Domain specific fields, otherwise, reader may not realize why there is an intentional empty line

Reply:
Fixed.